homeassistant/components/ezbeq/sensor.py

```diff
@@ -0,0 +1,77 @@
+"""Sensor platform for the ezbeq Profile Loader integration."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+import logging
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import EzBEQConfigEntry
+from .const import CURRENT_PROFILE, STATE_UNLOADED
+from .coordinator import EzBEQCoordinator
+from .entity import EzBEQEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class EzBEQSensorEntityDescription(SensorEntityDescription):
+    """Describe EzBEQ sensor entity."""
+
+    value_fn: Callable[[EzBEQCoordinator, str], StateType]
+
+
+SENSORS: tuple[EzBEQSensorEntityDescription, ...] = (
+    EzBEQSensorEntityDescription(
+        key=CURRENT_PROFILE,
+        value_fn=lambda coordinator, device_name: coordinator.client.get_device_profile(
```

Comment:
What are the possible states? Are they custom?

Reply:
It will be a string of the movie title